front/tests/assets/trainSchedule/train_schedules.json

```
@@ -92,7 +92,7 @@
     "constraint_distribution": "MARECO",
     "speed_limit_tag": "HLP",
     "power_restrictions": [],
-    "options": { "use_electrical_profiles": false }
+    "options": { "use_electrical_profiles": true, "stop_at_next_signal": false }
```

From previous value, it looks like a bad copy-paste:
```suggestion
    "options": { "use_electrical_profiles": false, "stop_at_next_signal": false }
```

---

oops, thanks for the spot 🎯